memegen/stores/image.py

```diff
@@ -60,7 +53,7 @@ def __init__(self, image):
             # is there no entry? should we query for one
             # or create a new one?
             if not self._word_models.get(word):
-                model = SA.query(WordModel).filter_by(id=word).first()
+                model = db.session.query(WordModel).filter_by(id=word).first()
```

Comment:
I like using `Query.filter()` instead because if can handle joins, inequalities and lots of other stuff that `filter_by` can't. I also think it looks nicer and involves less magic.

```python
db.session.query(WordModel).filter(WordModel.id == word).first()
```